### DISCUSSION TITLE
How to create a poster image download button like in a viewer editor

### DISCUSSION THREAD

**Discussion by bamiendev** [google/model-viewer#4518] [Q&A]
I'm creating a button to take a photo of a 3D model on my website but I'm having some problems:

+) Materials are hidden when:
material.setAlphaMode("MASK");
material.setAlphaCutoff(0.01);
I want to take photos of posters without hidden objects in the image.

+) I have a function js to get the parameters of the camera-orbit in the form:   deg   deg     m
How to image a 3D model according to the derivative of this function

I want my photos to look like:
![poster (4)](https://github.com/google/model-viewer/assets/116520283/a1ad8746-9bc9-474e-b929-bb34fd78e75b)
Instead of: 
![poster (4)](https://github.com/google/model-viewer/assets/116520283/f31e1849-43bd-423f-b0d7-fe7715b8ae10)

Thanks!

**Comment by bamiendev** [comment]
Sorry I want my photos to look like: 
![Uploading poster (3).png…]()
Instead of:
![poster (4)](https://github.com/google/model-viewer/assets/116520283/57c4ad03-f361-432d-8889-104f93508897)

**Comment by elalish** [comment]
All your images appear to be identical, so I'm not sure what you want to be different.

**Reply by bamiendev** [reply]
Thank you for your answer 
I was able to fix it
